Makefile

```diff
@@ -46,6 +46,10 @@ integration:
 build:
 	 $(BUILDCOMMAND) -o ${BINDIR}/okteto
 
+.PHONY: build-bin-image
+build-bin-image:
```

Comment:
should we add a manual CI job for this?

Reply:
or maybe use the CLI version as the tag for `okteto\bin` and build them both during release?